<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSSH</class>
 <widget class="QWidget" name="EditSSH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">EditVMess</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Host Key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="privateKeyPassphrase"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="hostKey"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>optional</string>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Private Key Passphrase</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>user</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="user"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="toolTip">
      <string>optional</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Private Key</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="password"/>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="privateKey"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="toolTip">
      <string>Defult: Random</string>
     </property>
     <property name="text">
      <string>Client Version</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="clientVersion"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hostKey</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
